cmd/integration/integration.go

```diff
@@ -955,6 +959,7 @@ type testFunc func(*client.Client)
 func addFlags(fs *pflag.FlagSet) {
 	fs.IntVar(
 		&maxConcurrency, "max-concurrency", -1, "Maximum number of tests to be run simultaneously. Unlimited if set to negative.")
+	fs.StringVar(&etcdServer, "etcd-server", "", "etcd server URL.")
```

Comment:
Or make it consistent with the other integration tests... weird to use a flag in one place and env vars in the other.